lib/desktop_screens/desktop_history/desktop_history.dart

```diff
@@ -356,6 +358,7 @@ class _DesktopHistoryScreenState extends State<DesktopHistoryScreen>
             onTap: () {
               setState(() {
                 _showSearchField = false;
+                historyProvider.setHistorySearchText = "";
               });
```

Comment:
Why are we setting setHistorySearchText as empty string on tap ?

Reply:
Because when we click on close button in the search bar, we need to reset the search results. 
This was not happening before on desktop